source/mir/ndslice/selection.d

```diff
@@ -947,8 +947,8 @@ auto byElement(size_t N, Range)(auto ref Slice!(N, Range) slice)
                 }
             }
 
-            static if (PureN == 1 && rangeHasMutableElements && !hasAccessByRef)
-            auto front(DeepElemType elem) @property
+            static if (PureN == 1 && isMutable!DeepElemType && !hasAccessByRef)
+            auto front()(E elem) @property
```

Comment:
Why does this have to be an empty template? (I vaguely remember having to do this myself, but I can't remember why...)